pkg/reconciler/cache/replication/replication_reconcile_test.go

```diff
@@ -48,49 +48,49 @@ func init() {
 func TestReconcileAPIExports(t *testing.T) {
 	scenarios := []struct {
 		name                                     string
-		initialLocalApiExports                   []runtime.Object
```

Comment:
nit: in the future, breaking spelling changes or other unrelated work into a separate commit aids in review

Reply:
ack